content/getting-started/multiple-projects-in-one-repository.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Multiple projects in one repository
 weight: 5
+draft: true
```

Comment:
Just to verify - was this set back to draft on purpose because this section will be reworded later?

Reply:
I set this to draft because I am now describing it getting-started.md. I figure it needn't be a separate page.

Reply:
OK, you can also delete it later if you don't intend to publish anymore. The history is always accessible via git